### DISCUSSION TITLE
(mpfs-axi4-stream-demo) Broken Link to bare metal SoftConsole Project

### DISCUSSION THREAD

**Discussion by ChrRudel** [polarfire-soc/polarfire-soc-documentation#26] [Applications and demos] [answered]
I am looking for the axi4-stream demo SoftConsole Project.
The provided link in the mpfs-axi4-stream-demo.md document seems to be broken.

https://github.com/polarfire-soc/polarfire-soc-bare-metal-examples/tree/main/driver-examples/fpga-ip/CoreAXI4DMAController

=> Leads to 404 error

friendly regards,
Christian

**Comment by hughbreslin** [comment, marked as answer]
Hi @ChrRudel 

thanks for pointing this out :) The documentation for this demo got mirrored early and the bare metal demo is going to be released as part of the .06 bare metal examples release. I'll respond here and let you know when its been mirrored. 

Cheers,
Hugh

**Reply by josh25219** [reply]
Hi Hugh,
I am also looking for the AXIstreaming Demo for a customer... Is it available?  I can't find it on Github anywhere.
Thanks,
Josh

**Comment by FPGA-forever** [comment]
Hi Christian,
In the mean time, there are some bare metal examples.
1)) GITHIB bare-metal page has examples:  https://github.com/polarfire-soc/polarfire-soc-bare-metal-examples
2)) App Note AC492 is on bare-metal apps, and is at this link:
https://ww1.microchip.com/downloads/aemDocuments/documents/FPGA/ApplicationNotes/ApplicationNotes/microchip_running_baremetal_user_applications_on_polarfire_soc_fpga_application_note_ac492_v2.pdf
